ipalib/plugable.py

```diff
@@ -494,6 +494,13 @@ def build_global_parser(self, parser=None, context=None):
         """
         Add global options to an optparse.OptionParser instance.
         """
+        def config_file_callback(option, opt, value, parser):
+            value = os.path.abspath(value)
+            if not os.path.isfile(value):
+                raise optparse.OptionValueError(
+                    "%s option '%s' is not a file" % (opt, value))
```

Comment:
1) Please use internationalized string here

2) instead of path.isfile(), I would use ipautil.file_exists() to be consistent with other parts

3) I don't like the error message, "-c option '/tmp/test.conf' is not a file" doesn't sound nice to me
IMO "%(filename)s: file not found" is better